src/app/components/icons/IconSearchBar.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { ChangeEvent, useEffect, useState, HTMLAttributes, useCallback } from 'react';
+import { Search } from '@material-ui/icons';
+import { useQueryString } from '../../hooks/useQueryString';
+// Material-UI Components
+import {
+    Checkbox,
+    FormControl,
+    Grid,
+    InputAdornment,
+    InputLabel,
+    ListItemText,
+    makeStyles,
+    MenuItem,
+    // OutlinedInput,
+    Select,
+    TextField,
+    useMediaQuery,
+    // Theme,
+} from '@material-ui/core';
+
+// import * as Colors from '@pxblue/colors';
+import { titleCase } from '../../shared';
+
+const useStyles = makeStyles(() => ({
+    searchBar: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+    searchField: {
+        // flex: '1 1 0px',
```

Comment:
Can this class be removed?

Reply:
Yes, I mean't to go through and do an unused class/var cleanup but forgot. I'll do that today.